pom.xml

```diff
@@ -71,6 +79,68 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
+            <plugin>
```

Comment:
Spring Boot already makes a fat-jar, why do you need these plugins?

Reply:
Removed [done]